test/json_reporter.js

```diff
@@ -0,0 +1,250 @@
+'use strict';
+
+var expect = require('chai').expect;
+var fs = require('fs');
+var JsonReporter = require('../lib/reporter/json_reporter.js');
+
+describe('json_reporter', () => {
+  describe('#constructor(opts)', () => {
+    it('sets defaults with no options given', () => {
+      let reporter = new JsonReporter();
+      expect(reporter.fileReports).to.be.empty;
+      expect(reporter.json).to.be.empty;
+    });
+  });
+
+  describe('#addFile(file, fileContent, items)', () => {
```

Comment:
This is covered by test/reporter.js, so I don't think we need it in this test suite.